format/testfiles/test.rego.formatted

```diff
@@ -228,9 +228,9 @@ declare2 := 2 {
 
 multi_line_with {
 	fn(1) with input.a as "a"
```

Comment:
I'd prefer if we got away from the one big for here. What do you think about splitting off the "with" related things into a new file?

Reply:
Sure!